src/common/Navigation/styled.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import { NavLink } from "react-router-dom";
+
+export const StyledNavigation = styled.nav`
+    width: 100%;
+    background-color: ${({ theme }) => theme.colors.navBackground};
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    display: flex;
+    justify-content: center;
+    padding: 10px;
+`;
+
+export const NavigationList = styled.ul`
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    width: max-content;
+    margin: 0;
+`;
+
+export const StyledCameraIcon = styled.img`
+    width: 32px;
+    height: 43px;
+    border: 2.5px solid ${({ theme }) => theme.colors.white};
+    margin: 0 10px;
+`;
+
+export const StyledTitle = styled.h1`
+    color: ${({ theme }) => theme.colors.navText};
+    position: inherit;
+    width: 168px;
+    height: 40px;
+    left: 52px;
+    margin: 0;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 40px;
+    letter-spacing: -1.5px;
+    text-transform: capitalize;
+`;
+
+export const StyledLabel = styled.label`
+    background-color: ${({ theme }) => theme.colors.white};
+    border: 1px solid ${({ theme }) => theme.colors.mystic};
+    border-radius: 33px;
+    position: inherit;
+    width: 30%max-content;
+    height: 40px;
+    display: flex;
+    align-items: center;
+    left: 0%;
+    right: 0%;
+    top: 0%;
+    bottom: 0%;
+`;
+
+export const StyledInput = styled.input`
```

Comment:
Jeżeli chcesz dodać obrazek lupy to możesz to zrobić w CSS 

```css
background: url(<<tutaj wstawiasz obrazek>>) no-repeat scroll 7px 7px;
padding-left:30px;
```

Reply:
ja ten obrazek próbowałam na różne sposoby dodawać, on i tak nie pokaże się w inpucie, dopóki te wartości pewnie nie będą wpisane (plceholder, on click i value), bo przecież jest zamknięty znacznik na końcu, przeglądałam też w innych projektach MB i niczym się ten kod nie różni do tej pory